src/main/java/seedu/simplykitchen/storage/JsonSerializableSimplyKitchenInventory.java

```diff
@@ -0,0 +1,60 @@
+package seedu.simplykitchen.storage;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonRootName;
+
+import seedu.simplykitchen.commons.exceptions.IllegalValueException;
+import seedu.simplykitchen.model.ReadOnlySimplyKitchenInventory;
+import seedu.simplykitchen.model.SimplyKitchenInventory;
+import seedu.simplykitchen.model.food.Food;
+
+/**
+ * An Immutable SimplyKitchenInventory that is serializable to JSON format.
+ */
+@JsonRootName(value = "simplyKitchen")
+class JsonSerializableSimplyKitchenInventory {
```

Comment:
Will "JsonSerializableFoodInventory" be better?

Reply:
Sure thing, I'll make the changes.